README.md

```diff
@@ -24,7 +24,7 @@ which define patterns and actions:
     }
 
 Metrics are exported for scraping by a collector as JSON or Prometheus format
-over HTTP, or can be periodically sent to a collectd, StatsD, or Graphite
+over HTTP, or can be periodically sent to a collected, StatsD, or Graphite
```

Comment:
collectd is the name of the program.  We could wrap these in `` to highlight them, though.